### PR TITLE
Try to make it works with ARM hosts

### PR DESCRIPTION
**Pull request by nledez**

### CODE REVIEW

**Comment by frankrousseau** [comment]
Hi @nledez, Thank you for this PR. But nodejs version of Debian is often way too old. Are you sure that your problem works in any case?

**Comment by nledez** [comment]
@frankrousseau I test it with a Ubuntu 14.04 the default distribution purpose me on the ARM provider.

**Comment by frankrousseau** [comment]
So, your changes won't work on Debian 6, 7 and Ubuntu 12.04. We should detect this distribution and launch a build with the `--without-snapshot` flag during the configuration step.

File: fabfile.py [hunk 1]
```diff
@@ -224,6 +230,17 @@ def install_node10():
         sudo('ln -s /opt/node/bin/npm /usr/local/bin/npm')
         sudo('ln -s /opt/node/bin/npm /usr/bin/npm')
 
+    elif is_arm:
+        distrib = system.distrib_id()
+        if distrib in ['Debian', 'Ubuntu']:
+            sudo('DEBIAN_FRONTEND=noninteractive apt-get --quiet update')
+            sudo('DEBIAN_FRONTEND=noninteractive apt-get install --assume-yes nodejs npm')
+            sudo('ln -s /usr/bin/nodejs /usr/local/bin/node')
```
Comment (nono) [inline fabfile.py]: @nledez why not using the nodejs-legacy package?

**Comment by Kloadut** [comment]
See #144

**Pull request closed without merging**